src/components/home/SideBarButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { FunctionComponent } from "react";
```

Comment:
I am starting to see there might be different button variations..... i wonder if it's a good idea to talk to design about giving us a list of potential button looks and we can have a single button component that takes in for example a "palette" prop that changes the looks of it to try to standardize buttons for our app.  I bring this up bc i saw some differences in the signup flow between mobile and desktop. Thoughts?